src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figure = new Figure[6];
+        int i = 0;
+        for (; i < figure.length / 2; i++) {
+            figure[i] = new FigureSupplier().getRandomFigure();
+            System.out.println(figure[i].draw());
+        }
+        while (i != figure.length) {
+            figure[i] = new FigureSupplier().getDefaultFigure();
+            System.out.println(figure[i].draw());
+            i++;
+        }
+    }
+
+}
+
```

Comment:
empty line

Reply:
not fixed, there should be only 1 empty line at the end, you have 2